intermediate_html_css/intermediate_css_concepts/advanced_selectors.md

```diff
@@ -80,7 +80,7 @@ Phrased another way, the child selector will select an element that is one level
 Finally, if we want to select all of the siblings following an element and not just the first one, we can use the general sibling combinator `~`.
 
 ```css
-/* This rule will select all of .group1's siblings - in this case the 2nd and 3rd .child divs */
+/* All of .group1's siblings - in this case the 2nd and 3rd .child divs, will get selected by this */
```

Comment:
Necessary to specify, otherwise we're describing `.group1 ~ *` by mistake.
```suggestion
/* All of .group1's div siblings - in this case the 2nd and 3rd .child divs, will get selected by this */
```